src/trufflesuite_ganachecli/mod.rs

```diff
@@ -1,19 +1,56 @@
 use std::borrow::Cow;
 
-use testcontainers::{core::WaitFor, Image};
+use testcontainers::{
+    core::{ContainerPort, WaitFor},
+    Image,
+};
 
 const NAME: &str = "trufflesuite/ganache-cli";
 const TAG: &str = "v6.1.3";
 
+/// Port that the [`Ganache CLI`] container has internally.
+/// Can be rebound externally via [`testcontainers::core::ImageExt::with_mapped_port`]
+///
+/// [Ganache CLI]: https://github.com/trufflesuite/ganache
+pub const GANACHE_CLI_PORT: ContainerPort = ContainerPort::Tcp(8545);
+
+/// # Module to work with the [`Ganache CLI`] inside of tests.
+///
+/// Starts an instance of Meilisearch.
+/// This module is based on the official [`trufflesuite/ganache-cli` docker image] documented in the [documentation].
+///
+/// # Example
+/// ```
+/// use testcontainers_modules::{testcontainers::runners::SyncRunner, trufflesuite_ganachecli};
+/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT
```

Comment:
So far for being smart and not breaking out an IDE for this

```suggestion
/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT;
```